docs/strongly-typed-errors.md

```diff
@@ -0,0 +1,265 @@
+---
+id: strongly-typed-errors
```

Comment:
I think you need to delete this file now?

Reply:
will do